src/descriptors/structs.json

```diff
@@ -177,5 +177,68 @@
       }
     ],
     "supertype": "Forecast"
+  },
+  {
+    "struct_name": "CostCoefficient",
+    "docstring": "A forecast for cost function data field in a Component.",
+    "fields": [
+      {
+        "name": "label",
+        "data_type": "String",
+        "comment": "label of component parameter forecasted"
+      },
+      {
+        "name": "break_points",
+        "data_type": "Int64",
```

Comment:
There is inconsistency in our code base on Int and Int64.  If you don't actually care about the number of bits (in this case you almost certainly don't), just use Int and let Julia use the native type for the system.